.github/workflows/build.yml

```diff
@@ -0,0 +1,31 @@
+name: NodeJS with Webpack
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16.x, 17.x]
```

Comment:
It would be good to leave a comment that the intention is to support the latest regular and the latest LTS versions. Right now it looks a little bit arbitrary. (YAML supports comments just fine.)